Source/WebGPU/WebGPU/HardwareCapabilities.mm

```diff
@@ -96,6 +96,7 @@ static HardwareCapabilities apple3(id<MTLDevice> device)
     auto baseCapabilities = WebGPU::baseCapabilities(device);
 
     baseCapabilities.supportsNonPrivateDepthStencilTextures = true;
+    baseCapabilities.canPresent10A2PixelFormats = false;
```

Comment:
where does this come from? I thought all devices since Apple3 supported RGB10A2: https://developer.apple.com/metal/Metal-Feature-Set-Tables.pdf

Reply:
https://developer.apple.com/documentation/quartzcore/cametallayer/1478155-pixelformat?language=objc